Assets/Scripts/ProceduralGeneration.cs

```diff
@@ -38,16 +53,21 @@ public class ProceduralGeneration : MonoBehaviour
     public static bool FinishedGenerating = false;
     private int maxEntities;
 
+    private float canisterRadius;
+
     private void Awake()
     {
-        asteroidsCollection = GameObject.FindGameObjectWithTag("Asteroids");
+        asteroidsCollection = GameObject.FindGameObjectWithTag("AsteroidCollection");
+        medicalCanisterCollection = GameObject.FindGameObjectWithTag("MedicalCanisterCollection");
+        
+        canisterRadius = medicalCanisterPrefab.GetComponent<Renderer>().bounds.size.x;
```

Comment:
Isn't this the edge length, not the radius? So it should be `bounds.size.x / 2f`

Reply:
Actually I don't think you even want the radius here, right? So can you rename it to `canisterSize`?